lib/constants/constants.go

```diff
@@ -0,0 +1,8 @@
+package constants
+
+const (
+	KeymasterAdminPortNumber   = 6920
+	KeymasterServicePortNumber = 443
+
+	CertmonPortNumber = 6921
```

Comment:
certmonPortNumber? ... I see that you (just like we had discussed) are using the controlPort in a separate path.
I would drop this file

Reply:
`CertmonPortNumber` will be used by `keymaster-certmond`, which is coming in a followup PR. I'll send that once you've added the `Publish*()` method calls, so I can test.